update_lzma_config_header.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# This is a helper script to update the config header for LZMA (i.e. config.lzma-android.h, 
+# config.lzma-ios-arm64.h, ...). Every time the version number of xz is bumped the corresponding 
+# config headers should be updated
+
+set -euxo pipefail
+
+# Provide rules_boost folder as a command line argument
+if [ "$#" -ne 1 ]; then
+    echo "Wrong number of parameters detected"
+    echo "Usage: $0 <rules_boost_dir> (e.g. ./update_lzma_config_header.sh ~/Downloads/rules_boost)"
+    exit 1
+fi
+
+rules_boost_dir=$1 # First command line argument e.g. ~/Downloads/rules_boost
+
+# Create a temporary directory
+tmpdir=$(mktemp -d)
+
+# Get rid of temporary files when script exits
+trap "rm -rf $tmpdir" EXIT
+
+
+# Download, and untar xz-5.4.0
+cd "$tmpdir"
+curl -sL https://cfhcable.dl.sourceforge.net/project/lzmautils/xz-5.4.0.tar.gz --output xz-5.4.0.tar.gz
+tar -xf xz-5.4.0.tar.gz
+
+# Switch to xz-5.4.0 directory
+cd xz-5.4.0
+
+CC="clang -arch arm64 \
+-isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS.sdk" \
```

Comment:
Agreed, especially since this path can take another form if people only have Apple command line tools installed.

For clarity, the change would be changing the `-isysroot` line to `-isysroot $(xcrun --sdk iphoneos --show-sdk-path) \`

Reply:
Done.